specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,3921 @@
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/dhcpId"
+          },
+          {
+            "name": "workloadNetworkDhcp",
+            "description": "NSX DHCP",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkDhcp"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkDhcp"
+            }
+          },
+          "202": {
+            "description": "Accepted. The patch will complete asynchronously"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_UpdateDhcp": {
+            "$ref": "./examples/WorkloadNetworks_UpdateDhcpConfigurations.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "WorkloadNetworks_DeleteDhcp",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Delete dhcp by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/dhcpId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The delete has succeeded"
+          },
+          "202": {
+            "description": "Accepted. The delete will complete asynchronously"
+          },
+          "204": {
+            "description": "No Content. No DHCP configuration to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_DeleteDhcp": {
+            "$ref": "./examples/WorkloadNetworks_DeleteDhcpConfigurations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/default/gateways": {
```

Comment:
/gateways needs an individual resource GET as well to retrieve the singleton gateway resource  (i.e. /gateways/{gatewayName})

Reply:
@ChristianPomales

Reply:
operationId WorkloadNetworks_GetGateway in 3821635